icelake/src/types/arrow/to_arrow.rs

```diff
@@ -114,7 +114,7 @@ impl TryFrom<types::Primitive> for ArrowDataType {
                 Ok(ArrowDataType::Decimal128(precision, scale as i8))
             }
             types::Primitive::Date => Ok(ArrowDataType::Date32),
-            types::Primitive::Time => Ok(ArrowDataType::Time32(TimeUnit::Microsecond)),
+            types::Primitive::Time => Ok(ArrowDataType::Time64(TimeUnit::Microsecond)),
```

Comment:
Should we also modify TimeUnit?

Reply:
Nope, https://docs.rs/arrow-array/48.0.0/src/arrow_array/types.rs.html#202-207

Reply:
Oh, sorry, my bad.